client/reducers/user.js

```diff
@@ -2,16 +2,37 @@ import axios from 'axios';
 import {setError} from './index'
 
 // ACTION TYPES
+const GET_USER = 'GET_USER';
+const UPDATE_USER_INFO = 'UPDATE_USER_INFO';
+const DELETE_USER = 'DELETE_USER';
 const SET_USER = 'SET_USER'
 
 // ACTION CREATORS
+export const getUserInfo = userInfo => {
```

Comment:
I think they wanted us to change getUserInfo and userInfo to getUser and user.

Reply:
@nicoleivanov i changed action type , but left the creator so it doesn't conflict with the thunk we're exporting. Had to make sure they weren't the same variable names